README.md

```diff
@@ -164,6 +164,17 @@ class Store {
 }
 ```
 
+## MobX v6
+
+In order to become compatible with modern ES standards, decorators are not used by default in MobX v6. It still supports decorators, but they are not recommended for greenfield projects and require some extra setup as opposed to previous versions.
+[Read More](https://michel.codes/blogs/mobx6)
+
+- In order to use MobX 6 with decorators `makeObservable(this)` should be added to the constructor, and "useDefineForClassFields": true should be added to tsconfig.json.
```

Comment:
That's a good point. I think the whole migration guide could be included here: https://mobx.js.org/migrating-from-4-or-5.html

Reply:
good idea